core/dive.c

```diff
@@ -3268,11 +3268,11 @@ struct dive *merge_dives(struct dive *a, struct dive *b, int offset, bool prefer
 	res->when = dl ? dl->when : a->when;
 	res->selected = a->selected || b->selected;
 	merge_trip(res, a, b);
-	MERGE_TXT(res, a, b, notes);
-	MERGE_TXT(res, a, b, buddy);
-	MERGE_TXT(res, a, b, divemaster);
+	MERGE_TXT(res, a, b, notes, "\n");
```

Comment:
so the notes this way would be separated with `\n` instead of being `(%s) or (%s)`. i think that either Linus or Dirk originally made it like so, so i'm going to leave it to them for approval.

@dirkhh

Reply:
This is impossible to do "right" - it's always a matter of taste. Linus picked the ( ) ( ) approach and I never really loved it, but at least it made clear that this was the result of a merge.
For buddy and dive master I think it makes sense to just concatenate them. For notes, though... how about
```
Notes 1
Notes 1
--
Notes 2
Notes 2
```
I.e., make the separator "\n--\n" ?

Reply:
@dirkhh 
For me everything is ok which works nicely with plain text notes and html notes from the planner.
The "\n--\n" thing definitively will work nicely as well so ok for me to use this.